src/cellfinder_core/tools/prep.py

```diff
@@ -16,61 +16,77 @@
 from cellfinder_core.download.download import amend_cfg
 from cellfinder_core.tools.source_files import source_custom_config_cellfinder
 
+home = Path.home()
+DEFAULT_INSTALL_PATH = home / ".cellfinder"
+
 
 def prep_classification(
-    trained_model, model_weights, install_path, model, n_free_cpus
-):
+    trained_model: Optional[os.PathLike],
+    model_weights: Optional[os.PathLike],
+    model_name: str,
+    n_free_cpus: int,
+    install_path: os.PathLike = DEFAULT_INSTALL_PATH,
+) -> Path:
     n_processes = get_num_processes(min_free_cpu_cores=n_free_cpus)
     prep_tensorflow(n_processes)
     model_weights = prep_models(
-        trained_model, model_weights, install_path, model
+        trained_model, model_weights, model_name, install_path
     )
 
     return model_weights
 
 
 def prep_training(
-    n_free_cpus, trained_model, model_weights, install_path, model
-):
+    n_free_cpus: int,
+    trained_model: Optional[os.PathLike],
+    model_weights: Optional[os.PathLike],
+    model_name: str,
+    install_path: os.PathLike = DEFAULT_INSTALL_PATH,
+) -> Path:
     n_processes = get_num_processes(min_free_cpu_cores=n_free_cpus)
     prep_tensorflow(n_processes)
     model_weights = prep_models(
-        trained_model, model_weights, install_path, model
+        trained_model, model_weights, model_name, install_path=install_path
     )
     return model_weights
 
 
-def prep_tensorflow(max_threads):
+def prep_tensorflow(max_threads: int) -> None:
     tf_tools.set_tf_threads(max_threads)
     tf_tools.allow_gpu_memory_growth()
 
 
-def prep_models(trained_model, model_weights, install_path, model):
+def prep_models(
+    trained_model_path: Optional[os.PathLike],
+    model_weights_path: Optional[os.PathLike],
+    model_name: str,
+    install_path: os.PathLike = DEFAULT_INSTALL_PATH,
+) -> Path:
     # if no model or weights, set default weights
-    if trained_model is None and model_weights is None:
+    if model_weights_path is None:
         logging.debug("No model or weights supplied, so using the default")
```

Comment:
Does this error message need to change as a result?

Reply:
Yes, thanks for catching